packages/ag-charts-enterprise/src/features/zoom/zoom.ts

```diff
@@ -86,17 +93,19 @@ export class Zoom extends _ModuleSupport.BaseModuleInstance implements _ModuleSu
         this.zoomManager = ctx.zoomManager;
         this.updateService = ctx.updateService;
 
+        const interactionOpts = { bypassPause: ['animation' as const] };
         this.destroyFns.push(
-            ctx.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event)),
-            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event)),
-            ctx.interactionManager.addListener('drag-end', () => this.onDragEnd()),
-            ctx.interactionManager.addListener('wheel', (event) => this.onWheel(event)),
-            ctx.interactionManager.addListener('hover', () => this.onHover()),
+            ctx.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event), interactionOpts),
+            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event), interactionOpts),
+            ctx.interactionManager.addListener('drag-end', () => this.onDragEnd(), interactionOpts),
+            ctx.interactionManager.addListener('wheel', (event) => this.onWheel(event), interactionOpts),
+            ctx.interactionManager.addListener('hover', () => this.onHover(), interactionOpts),
             ctx.chartEventManager.addListener('axis-hover', (event) => this.onAxisHover(event)),
             ctx.layoutService.addListener('layout-complete', (event) => this.onLayoutComplete(event))
         );
 
-        // Add layout listener
+        this.anchorPoints = new ZoomAnchorPoints();
```

Comment:
Nit: Maybe just initialise this inline with the field declaration to reduce clutter? (Same for `axisDragger` below?)